docs/windows.md

```diff
@@ -5,21 +5,30 @@
 To setup a development environment in Windows:
 1) Go to http://www.msys2.org/ and download the x86_64 installer
 1) Follow the instructions on the page for setting up the basic environment
-1) Run ``C:\msys64\mingw64.exe`` - a terminal window should pop up
+1) Run `C:\msys64\mingw64.exe` - a terminal window should pop up
 ```bash
 $ pacman -Suy
-$ pacman -S git mingw-w64-x86_64-gcc mingw-w64-x86_64-gtk3 \
-mingw-w64-x86_64-pkg-config mingw-w64-x86_64-cairo \
-mingw-w64-x86_64-gobject-introspection mingw-w64-x86_64-python \
-mingw-w64-x86_64-python-gobject mingw-w64-x86_64-python-cairo \
-mingw-w64-x86_64-python-pip
+$ pacman -S git \
+    mingw-w64-x86_64-gcc \
+    mingw-w64-x86_64-gtk3 \
+    mingw-w64-x86_64-pkg-config \
+    mingw-w64-x86_64-cairo \
+    mingw-w64-x86_64-gobject-introspection \
+    mingw-w64-x86_64-python \
+    mingw-w64-x86_64-python-gobject \
+    mingw-w64-x86_64-python-cairo \
+    mingw-w64-x86_64-python-pip \
+    mingw-w64-x86_64-python-setuptools \
```

Comment:
I updated the list to include setuptools, since that was also in CI. What's the difference between a local installation and a CI installation then?

Reply:
Ya, I'm not sure what is different in CI either, I can try to do a clean MSYS2 install and test it out this weekend.

Reply:
It looks like in CI that it is installing a few packages by using setup.py instead of building a wheel:

```
Using legacy 'setup.py install' for msgpack, since package 'wheel' is not installed.
Using legacy 'setup.py install' for pyrsistent, since package 'wheel' is not installed.
```
This doesn't seem to be slowing things down, but I think building the wheel would be preferred.

Reply:
Agreed. :+1: